migrations/202106301200_added_allowing_email_auth.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE tenants ADD COLUMN is_allowing_email_auth BOOLEAN;
+UPDATE tenants SET is_allowing_email_auth = TRUE;
```

Comment:
There was a migration file created on the previous PR with a date before this one, so we need this one to be moved to `20210720...`

Reply:
Yes, I saw the `..._add_locale_field` that came right after this one :sweat_smile: 
Renamed to `202107211126_...`